mail-mta/protonmail-bridge-bin/protonmail-bridge-bin-1.0.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit rpm
+
+DESCRIPTION="Serves ProtonMail to IMAP/SMTP clients"
+HOMEPAGE="https://protonmail.com/bridge/"
+SRC_URI="https://protonmail.com/download/${P/-bin/}-1.x86_64.rpm"
+
+RESTRICT="mirror"
+
+LICENSE="MIT protonmail-bridge-EULA"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="
+	app-crypt/libsecret
+	dev-libs/glib
```

Comment:
Please make it `:2`, we used to have glib-1 in the same slot, and it's kinda tradiion to specify the slot here.

Reply:
done